scene/3d/baked_lightmap.cpp

```diff
@@ -1463,6 +1511,8 @@ void BakedLightmap::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "quality", PROPERTY_HINT_ENUM, "Low,Medium,High,Ultra"), "set_bake_quality", "get_bake_quality");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "bounces", PROPERTY_HINT_RANGE, "0,16,1"), "set_bounces", "get_bounces");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "use_denoiser"), "set_use_denoiser", "is_using_denoiser");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "use_ldr"), "set_use_ldr", "is_using_ldr");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "use_grayscale"), "set_use_grayscale", "is_using_grayscale");
```

Comment:
I wonder if those properties should be inverted to be "affirmative" rather than negative. In other words, turn `use_ldr` into `use_hdr` and `use_grayscale` with `use_color` (and invert their default values). This would be consistent with `use_denoiser` where enabling it results in improved quality.

Reply:
+1 for use_hdr instead of use_ldr, but I'd keep use_grayscale as is.

Reply:
I like the idea of keeping consistency with `use_denoiser`, so I went with both `use_hdr` and `use_color`.